docker-compose.yaml

```diff
@@ -37,6 +37,7 @@ services:
       - ${NODE_PORT}:8000
     volumes:
       - /etc/localtime:/etc/localtime:ro
+      - .:/app
```

Comment:
is there a reason this is being mounted here?

Reply:
No, is gone